performance-tests/perf-test-commits.sh

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+PERF_TESTS_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+source "${PERF_TESTS_DIR}"/../scripts/common.sh
+
+GREEN='\033[0;32m'
+NOCOLOR='\033[0m'
+
+FIND_COMMAND="find"
+if [ "$OS" = 'Darwin' ] && resolveCommand gfind ; then
+  FIND_COMMAND=gfind
+  ENABLE_REGEX="-regextype posix-extended"
+elif [ "$OS" = 'Darwin' ] ; then
+  # for BSD find
+  ENABLE_REGEX="-E"
+else
+  # for gnu find
+  ENABLE_REGEX="-regextype posix-extended"
+fi
+
+COMMITS=( "$@" )
+SHORT_COMMITS=( )
+
+#Cross platform temp directory creation based on https://unix.stackexchange.com/a/84980
+CHECKOUT_DIR=${CHECKOUT_DIR:=$(mktemp -d -t kroxyliciousPerfTestCheckout.XXXXXX 2>/dev/null || mktemp -d -t 'kroxyliciousPerfTestCheckout')}
```

Comment:
You are relying on gfind, common relies on gsed already, so relying on gmktemp is probably okay.

Reply:
mmmm I wasn't intending to depend on `gfind` but maybe we should just make that call and use the gnu tools. 

What do you think @robobario @gracegrimwood @tombentley ? Is it a reasonable proposition to ask Mac OS users to install gnu core utils to run our scripts?

Reply:
Your approach is sound.